pages/approval.tsx

```diff
@@ -24,6 +25,42 @@ const getAssessments = (taskNumber: number) => {
   }
 };
 
+const convertToNumber = (n: string | number) => {
+  if ( typeof n == "number") {
+    return n;
+  }
+  return parseInt(n);
+};
+
+// returns the each assessment with potentially inconsistent values
+function filterAssessments(assessments: AssessmentType[]): ApprovalType[] {
+  const uniqueAssessments: AssessmentType[] = assessments.sort((a:AssessmentType, b:AssessmentType) => {
+    return b.candidateId - a.candidateId
+  })
+    .filter((assessment, index, array) => { return !index || assessment.candidateId != array[index - 1].candidateId;
```

Comment:
Not quite sure what happens here. Commented so you can explain it to me later 👯‍♀️